README.md

```diff
@@ -2,11 +2,11 @@
 # WHOIS
 
 Publisher: Splunk  
-Connector Version: 2.2.2  
+Connector Version: 2.2.3  
 Product Vendor: Generic  
 Product Name: Whois  
 Product Version Supported (regex): ".\*"  
-Minimum Product Version: 6.1.1  
+Minimum Product Version: 6.2.1  
```

Comment:
I think the new "min_phantom_version" is 6.2.2

Reply:
```suggestion
Minimum Product Version: 6.2.2  
```